src/main/kotlin/org/rust/lang/core/stubs/index/RsExternCrateReexportIndex.kt

```diff
@@ -31,7 +30,8 @@ class RsExternCrateReexportIndex : StringStubIndexExtension<RsExternCrateItem>()
 
         fun index(stub: RsExternCrateItemStub, sink: IndexSink) {
             val externCrateItem = stub.psi
-            if (!externCrateItem.isPublic) return
+            val isPublic = externCrateItem?.vis != null
```

Comment:
Let's be more precise here and check that `visRestriction` is also null here

Reply:
This done in the same manner as was fixed in `RsModuleIndex` and also follows `RsVisibility.isPublic` logic (that returns true if there is any visibility modifier). If we want to change it, I think it should be another PR

Reply:
Agree